scripts/core/editor3/components/annotations/AnnotationInput.jsx

```diff
@@ -62,21 +63,33 @@ export class AnnotationInput extends Component {
     }
 
     componentDidMount() {
+        const annotationTypes = ng.get('metadata').values.annotation_types;
+
+        if (annotationTypes) {
+            this.setState({
+                annotationTypes: annotationTypes,
+                type: annotationTypes[0],
+            });
+        }
+
         $('.annotation-input textarea').focus();
     }
 
     render() {
         const {onCancel} = this.props;
-        const {type} = this.state;
+        const {type, annotationTypes} = this.state;
 
         return (
             <div className="annotation-input">
                 <Dropdown open={true}>
                     <div className="sd-line-input sd-line-input--is-select">
```

Comment:
Is it possible to have no annotation types set in the CV? If yes, then I'm thinking we should hide this `div` when that happens.